src/blocks/weather.rs

```diff
@@ -38,6 +38,17 @@
 //! in which case they must be provided in the environment variables
 //! `OPENWEATHERMAP_API_KEY`, `OPENWEATHERMAP_CITY_ID`, `OPENWEATHERMAP_PLACE`.
 //!
+//! # met.no Options
+//!
+//! Key | Values | Required | Default
+//! ----|--------|----------|--------
+//! `name` | `metno`. | Yes | None
+//! `coordinates` | GPS latitude longitude coordinates as a tuple, example: `["39.2362","9.3317"]` | Required if `autolocate = false` | None
```

Comment:
Why did you decide to accept coordinates as strings? Why not just use `f64`?

Reply:
Same thing with altitude

Reply:
I was worried that it would be confusing to have different configuration standards for openweathermap and metno, and did not want to invalidate old config files by changing the definition for openweathermap. 

For altitude I have no explanation.

Reply:
@ammgws is this worth a breaking change?